internal/cli/atlas/atlas.go

```diff
@@ -102,3 +110,24 @@ func Builder() *cobra.Command {
 
 	return cmd
 }
+
+func newDeprecatedMessage() string {
+	if isMongoCLIManagedWithBrew() {
+		deprecatedMessage := `There’s a new, dedicated Atlas CLI available for Atlas users. Install the Atlas CLI to enjoy the same capabilities and keep getting new features. Run "brew install mongodb-atlas-cli" or visit https://dochub.mongodb.org/core/migrate-to-atlas-cli.
+
+`
+		return fmt.Sprintf("%s %s", "\u26A0", deprecatedMessage)
+	}
+
+	return "\u26A0 There’s a new, dedicated Atlas CLI available for Atlas users. Install the Atlas CLI to enjoy the same capabilities and keep getting new features: https://dochub.mongodb.org/core/migrate-to-atlas-cli.\n\n"
+}
+
+func isMongoCLIManagedWithBrew() bool {
+	cmd := exec.Command("brew", "--prefix", "mongocli")
+
+	if err := cmd.Start(); err != nil {
+		return false
+	}
+
+	return true
+}
```

Comment:
we show `brew install mongodb-atlas-cli` if the user is managing mongocli with brew

Reply:
can you use `	c, _ := homebrew.NewChecker(n.filesystem)
	isHb = c.IsHomebrew()` instead of `isMongoCLIManagedWithBrew` since it has cached info?

Reply:
got an update to check for OS type instead of brew. I have removed this part.